nautobot/extras/models/jobs.py

```diff
@@ -663,12 +663,15 @@ def enqueue_job(
             redirect_logger = get_logger("celery.redirected")
             proxy = LoggingProxy(redirect_logger, app.conf.worker_redirect_stdouts_level)
             with contextlib.redirect_stdout(proxy), contextlib.redirect_stderr(proxy):
-                job_model.job_task.apply(
+                eager_result = job_model.job_task.apply(
                     args=job_args, kwargs=job_kwargs, task_id=str(job_result.id), **job_celery_kwargs
                 )
 
             # copy fields from eager result to job result
             job_result.refresh_from_db()
+            job_result.result = eager_result.result
+            job_result.status = eager_result.status
+            job_result.traceback = eager_result.traceback
```

Comment:
These were removed in #4683 to fix a security vulnerability. This doesn't reintroduce that vulnerability does it?

Reply:
Hmm, good catch and good question. I'll admit I don't quite see how that particular change in that PR (https://github.com/nautobot/nautobot/pull/4683/commits/621bf8df9ed020caab3cea75857b396f83bed381)  is directly related to the vulnerability fix. At a glance I feel like we still need to re-add this code but maybe we need to explicitly sanitize the result and traceback as well? Will do some digging.

Reply:
Ok, the test added in that PR is failing now, so let me see what I need to do to fix it.